converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/Patient/PatientPipelineStep.cs

```diff
@@ -55,7 +64,23 @@ protected override async Task PrepareRequestImplementationAsync(FhirTransactionC
             throw new MissingRequiredDicomTagException(nameof(DicomTag.PatientID));
         }
 
-        var patientIdentifier = new Identifier(string.Empty, patientId);
+        //Patient system id is determined based on issuer id boolean
+        //If issuer id boolean is set to true, patient system id would be set to issuer of patient id (0010,0021)
+        //Other wise we will be using the patient system id configured during user provisioning
+        string patientSystemId = string.Empty;
```

Comment:
Should we simply have one config value like `PatientSystemId` with possible values like:
- `""` (empty, like today)
- `"1"` (a constant value like your `_patientSystemId`)
- `"(0010,0021)"` (a DICOM tag -- perhaps identified by the parentheses)

Reply:
But we're not developing the feature based on Geneva's UI are we? What is the most simple solution for users of the OSS? We can always adjust how the UI affects the settings, right?

Reply:
Had a discussion within the team today. We would be overloading the patientSystemId tag if we are using a single property. Also the idea is to make it simple for the user (Either enable the issuer id usind a boolean or disable the same and configure a patient system id.